apps/web-giddh/src/app/activity-logs/activity-logs.component.html

```diff
@@ -0,0 +1,78 @@
+<div *ngIf="isLoading">
+    <giddh-page-loader></giddh-page-loader>
+</div>
+<ng-container
+    *ngIf="!isLoading && dataSource.length > 1"
+    appTranslate
+    [file]="'activity-logs'"
+    (localeData)="localeData = $event"
+    (commonLocaleData)="commonLocaleData = $event"
+>
+    <div class="activity-container">
+        <div class="table-content">
+            <table mat-table [dataSource]="dataSource" class="mat-elevation-z8">
+                <!-- UserName Column -->
+                <ng-container matColumnDef="name">
+                    <th mat-header-cell *matHeaderCellDef>Name</th>
+                    <td mat-cell *matCellDef="let element">
+                        <span matTooltip="{{ element.user.email }}" matTooltipClass="">{{ element.user.name }}</span>
+                    </td>
+                </ng-container>
+
+                <!-- Date Column -->
+                <ng-container matColumnDef="time">
+                    <th mat-header-cell *matHeaderCellDef>Date</th>
+                    <td mat-cell *matCellDef="let element">
+                        <span matTooltip="{{ element.time2 }}" matTooltipClass=""> {{ element.time }}</span>
```

Comment:
Please check the matTooltip value is set to `element.time2`

Reply:
Yes sir , we will provide time on hover on date